hack/verify-bazel-generated.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+cd "${KUBE_ROOT}"
+# List go packages with the k8s:openapi-gen tag.
+files=$(grep --color=never -l '+k8s:openapi-gen=' \
```

Comment:
I copied the find from the makefile, but I'm not sure that there's a good way to keep it up to date. We do want to include `vendor/`.

Using the make process, if I `touch vendor/k8s.io/apimachinery/pkg/apis/meta/v1/doc.go` then run `make generated_files`, it will recreate the generated files under there. Evidently the source of truth for that stuff is `vendor/`...